source/shared/cpp/AdaptiveCardsSharedModel/AdaptiveCardsSharedModelUnitTest/MarkDownUnitTest.cpp

```diff
@@ -249,13 +249,13 @@ namespace AdaptiveCardsSharedModelUnitTest
         TEST_METHOD(Rule11_12Test_EscapeTest)
         {
             MarkDownParser parser("foo *\\**");
-            Assert::AreEqual<std::string>("<p>foo <em>*</em></p>", parser.TransformToHtml());
+            Assert::AreEqual<std::string>("<p>foo <em>\\</em>*</p>", parser.TransformToHtml());
 
             MarkDownParser parser2("foo **\\***");
-            Assert::AreEqual<std::string>("<p>foo <strong>*</strong></p>", parser2.TransformToHtml());
+            Assert::AreEqual<std::string>("<p>foo <strong>\\</strong>*</p>", parser2.TransformToHtml());
```

Comment:
```suggestion```
            Assert::AreEqual<std::string>(R"<p>foo <strong>\</strong>*</p>", parser2.TransformToHtml());
```